two-regions-app-lb/masterstack.yaml

```diff
@@ -0,0 +1,103 @@
+heat_template_version: 2016-04-08
+
+description: This template launches application instances in two regions behind a LoadBalancer.
+
+parameters:
+
+  app_port:
+    type: number
+    default: 80
+    description: Port used by the servers
+  lb_port:
+    type: number
+    default: 80
+    description: Port used by the load balancer
+  image:
+    type: string
+    description: Image used by instances
+    default: "Ubuntu 18.04 LTS sys11 optimized - 2018-08-13"
+  ssh_keys:
+    type: comma_delimited_list
+    description: SSH-Key injected into instances
+    default:
+      - '< SSH PUBLIC KEY GOES HERE>'
```

Comment:
Those lines, if left untouched, disappear automatically. I suspect that this is not intended but due to a quoting issue, hence i'd like to dig into this further.

Reply:
this is not a blocker, anyway